gk/main.c

```diff
@@ -1365,11 +1362,11 @@ process_cmds_from_mailbox(
 {
 	int i;
 	int num_cmd;
-	struct gk_cmd_entry *gk_cmds[GK_CMD_BURST_SIZE];
+	struct gk_cmd_entry *gk_cmds[gk_conf->mailbox_burst_size];
```

Comment:
Let's have a variable for the expression `gk_conf->mailbox_burst_size` as in the other patches.